packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -630,12 +628,6 @@ async function handler(
   }
 ) {
   const { userId } = req;
-  const userIp = getIP(req);
-
-  await checkRateLimitAndThrowError({
```

Comment:
We shouldn't call rate limit here as this function is called by recurring-event route many times.